api/src/commonMain/kotlin/dev/kord/cache/api/observables/Cache.kt

```diff
@@ -0,0 +1,33 @@
+package dev.kord.cache.api.observables
+
+interface Cache<T: Any> {
```

Comment:
this interface is now missing a `put` function, what would this then be used for?

Reply:
It's a common interface of both interfaces.
But as we are looking into this together I think it makes no use to separate both implementations.
My essential thought was sometimes the Key value may be redundant but the user can user a Long or another place holder.